modules/mysql/metadata.yaml

```diff
@@ -914,3 +920,12 @@ spec:
       - serviceusage.googleapis.com
       - sqladmin.googleapis.com
       - workflows.googleapis.com
+    providerVersions:
```

Comment:
Thanks for the change! Was this auto-generated?

Reply:
Yes! I think this is generated per @qz267 's PR.